git-workflows/skills/creating-pull-request/SKILL.md

```
@@ -1,387 +1,135 @@
-
-**Step 1: Get current branch**
+## Phase 1: Pre-flight Checks
 
-Tool: `mcp__git__git_branch`
-Parameters:
-- `repo_path`: Current working directory absolute path
-- `branch_type`: "local"
-Permission: Auto-approved (read-only MCP tool)
-Expected output: List of local branches with current branch indicated
-Capture: Current branch name (the branch marked as current)
+Verify prerequisites for PR creation.
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)